ToneMapper/TMGLViewController.m

```diff
@@ -68,16 +50,24 @@ - (void)viewDidAppear:(BOOL)animated {
   if (!self.glkView) {
```

Comment:
Now you set the glkView frame on <code>viewWillTransitionToSize:withCoordinator:</code> instead.